config/characterizeImage.py

```diff
@@ -1,8 +1,23 @@
 """
 DECam-specific overrides for CharacterizeImageTask
 """
+import os.path
+
+from lsst.meas.astrom import MatchOptimisticBConfig
+
+obsConfigDir = os.path.dirname(__file__)
+
+# Cosmic rays
 config.repair.cosmicray.nCrPixelMax = 100000
+config.repair.cosmicray.cond3_fac2 = 0.4
```

Comment:
Please add a short explanatory comment for where this and subsequent config overrides originate from (e.g., matching obs_subaru for ease of 1:1 comparisons, or something else?)